exchange.js

```diff
@@ -40,6 +36,22 @@ fetch(url)
   })
   .catch((error) => console.error("Error fetching exchange rates:", error));
 
+// функция для динамического добавления опций в выпадающем списке
+function fillCurrencyOptions() {
+  currencies.forEach((currency) => {
+    const option1 = document.createElement("option");
+    option1.value = currency;
+    option1.textContent = currency;
+    CURRENCY1.appendChild(option1);
```

Comment:
I guess CURRENCY1 is a selector? if so, let's rename CURRENCY1 -> CURRENCY_1_SELECTOR
the same for 2